android/test_app/make_assets_custom.py

```diff
@@ -5,10 +5,12 @@
 """
 
 import yaml
+
```

Comment:
This line is a seperator between first-party import and third-party. Before this PR, both `yaml` and `torchvision` were third-party packages, so they are grouped without a separating empty line.